cli/exec/flags.go

```diff
@@ -64,6 +64,25 @@ var flags = []cli.Flag{
 		Value:   "auto-detect",
 	},
 
+	//
+	// backend options for pipeline compiler
```

Comment:
better name suggestions are welcome :)

other candidate of mine would be "pipeline default settings" ...